docker/Dockerfile

```diff
@@ -3,6 +3,8 @@
 
 FROM ubuntu:20.04
 
+RUN set -e
```

Comment:
I think this isn't having any effect. I believe that this only affects the shell instance invoked by this particular `RUN` command, and doesn't affect any subsequent `RUN` commands.  I believe the repro below shows this, in that if the change in question worked, the repro below would have failed on line 4 instead of line 5.

```
gene@gene-dev:~/dev/tmp$ cat -n Dockerfile
     1  FROM bash:4.4
     2  SHELL ["/usr/local/bin/bash", "-c"]
     3  RUN set -e
     4  RUN ls jjj; true
     5  RUN set -e; ls jjj; true
gene@gene-dev:~/dev/tmp$ docker build .
[+] Building 1.3s (7/7) FINISHED                                                                           docker:default
 => [internal] load .dockerignore                                                                                    0.2s
 => => transferring context: 2B                                                                                      0.0s
 => [internal] load build definition from Dockerfile                                                                 0.1s
 => => transferring dockerfile: 140B                                                                                 0.0s
 => [internal] load metadata for docker.io/library/bash:4.4                                                          0.0s
 => [1/4] FROM docker.io/library/bash:4.4                                                                            0.0s
 => CACHED [2/4] RUN set -e                                                                                          0.0s
 => CACHED [3/4] RUN ls jjj; true                                                                                    0.0s
 => ERROR [4/4] RUN set -e; ls jjj; true                                                                             0.9s
------                                                                                                                    
 > [4/4] RUN set -e; ls jjj; true:
0.362 ls: jjj: No such file or directory
------
Dockerfile:5
--------------------
   3 |     RUN set -e
   4 |     RUN ls jjj; true
   5 | >>> RUN set -e; ls jjj; true
   6 |     
--------------------
ERROR: failed to solve: process "/usr/local/bin/bash -c set -e; ls jjj; true" did not complete successfully: exit code: 1
```